File: testprojects/scala_main/pipelines/LIVY_SCALA/code/src/main/scala/io/prophecy/pipelines/livy_scala/graph/Reformat_6.scala
package io.prophecy.pipelines.livy_scala.graph

import io.prophecy.libs._
import io.prophecy.pipelines.livy_scala.config.ConfigStore._
import io.prophecy.pipelines.livy_scala.config.Context
import io.prophecy.pipelines.livy_scala.udfs.UDFs._
import io.prophecy.pipelines.livy_scala.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_6 {

  def apply(context: Context, in: DataFrame): DataFrame = {
    val Config = context.config
    in.select(
      concat(col("year"),
             lit(Config.c_array(0).car_short),
             lit(Config.c_record.cr_double)
      ).as("year"),
      col("industry_code_ANZSIC"),
      col("industry_name_ANZSIC"),
      col("rme_size_grp"),
      col("variable"),
      col("value"),
      col("unit")
    )
  }

}

File: testprojects/master_python_mainone/pipelines/PYTHON_DEP_MANAGEMENT_ALL/code/job/graph/Reformat_6.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from job.config.ConfigStore import *
from job.udfs.UDFs import *

def Reformat_6(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(col("`c- short`").alias("c- short"), col("`c  - int`").alias("c  - int"))
